app/models/service_template.rb

```diff
@@ -206,19 +206,22 @@ def create_service(service_task, parent_svc = nil)
     svc
   end
 
+  def self.public_service_templates
```

Comment:
This should be a `scope` and moved up with the other scopes here: https://github.com/ManageIQ/manageiq/blob/master/app/models/service_template.rb#L73-L76

Should this logic be incorporated into the existing `displayed` scope?